### PR TITLE
Merge dev to master to follow AixLib update

### PR DESCRIPTION
**Pull request by MartinRaetz**
solves #688

### CODE REVIEW

**Comment by MartinRaetz** [comment]
I suggest to "squash and merge" - objections?

File: teaser/logic/buildingobjects/calculation/aixlib.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def __init__(self, parent):
         self.file_set_t_cool = "TsetCool_" + self.parent.name + ".txt"
         self.file_ahu = "AHU_" + self.parent.name + ".txt"
         self.file_internal_gains = "InternalGains_" + self.parent.name + ".txt"
-        self.version = "0.9.1"
+        self.version = "0.10.7"
```
Comment (DaJansenGit) [inline teaser/logic/buildingobjects/calculation/aixlib.py:64]: shouldn't this be raised to 1.0.0?

File: teaser/logic/buildingobjects/calculation/ibpsa.py [hunk 2]
```diff
@@ -36,8 +36,8 @@ def __init__(self, parent):
         self.parent = parent
         self.file_internal_gains = "InternalGains_" + self.parent.name + ".mat"
         self.version = {
-            "AixLib": "0.9.1",
-            "Buildings": "6.0.0",
+            "AixLib": "0.10.7",
```
Comment (DaJansenGit) [inline teaser/logic/buildingobjects/calculation/ibpsa.py:39]: should be 1.0.0

**Review by DaJansenGit — CHANGES_REQUESTED** [review]
Some minor things. Mostly version raise of aixlib version

File: teaser/logic/buildingobjects/calculation/ibpsa.py (hunk 2 above)
Comment (MartinRaetz) [reply teaser/logic/buildingobjects/calculation/ibpsa.py:39]: correct!

File: teaser/data/input/inputdata/MaterialTemplates.json [hunk 3]
```diff
@@ -7020,7 +7020,7 @@
     },
     "6cb62e70-3a43-11e7-a790-2cd444b2e704": {
         "name": "air",
-        "density": 1184.0,
+        "density": 1.184,
```
[the remark this answers is not in the document]
Comment (MartinRaetz) [reply teaser/data/input/inputdata/MaterialTemplates.json:7023]: I think this was faulty for Air. Though this is a change done in one of the branches merged into the development. I think the development is correct.

File: teaser/data/output/aixlib_output.py [hunk 4]
```diff
@@ -5,6 +5,7 @@
 from mako.template import Template
 from mako.lookup import TemplateLookup
 import teaser.logic.utilities as utilities
+import shutil
```
[the remark this answers is not in the document]
Comment (MartinRaetz) [reply teaser/data/output/aixlib_output.py:8]: Thank you for noticing! I changed all occurences I could found.

File: teaser/logic/buildingobjects/calculation/aixlib.py (hunk 1 above)
Comment (MartinRaetz) [reply teaser/logic/buildingobjects/calculation/aixlib.py:64]: Thank you! Correct!

**Review by MartinRaetz — COMMENTED** [review]
Some changes have been due to prior branches merged to the development. 
Thank you for noticing the improvements!

**Review by DaJansenGit — APPROVED** [review]

**Comment by DaJansenGit** [comment]
Thanks for the changes! :)

**Pull request merged**